Doc/library/socket.rst

```diff
@@ -200,6 +200,23 @@ created.  Socket addresses are represented as follows:
 
   .. versionadded:: 3.8
 
+- :const:`IPPROTO_UDPLITE` is a variant on UDP which allows you to specify
+  what portion of a packet to cover with the checksum. It adds the methods
```

Comment:
Please rephrase.
`IPPROTO_UDPLITE` doesn't `add the methods` but allows to use `UDPLITE_SEND_CSCOV` / `UDPLITE_RECV_CSCOV` in `setsockopt`

Reply:
Does the new version make this better in your eyes?